### ISSUE TITLE
pod 'DatadogSDKCrashReporting' is missing as of 1.7.3

### ISSUE THREAD

**Issue by dballance** [GetDutchie/datadog_flutter#99]
#95 does not add `pod 'DatadogSDKCrashReporting'` as a dependency. See: https://github.com/DataDog/dd-sdk-ios/issues/778

This causes a crash during builds:

```
Swift Compiler Error (Xcode): No such module 'DatadogCrashReporting'
/opt/flutter/.pub-cache/hosted/pub.dartlang.org/datadog_flutter-1.7.3/ios/Classes/SwiftDatadogFlutterPlugin.swift:5:7
```

**Comment by tshedor** [comment]
Just when I think all the bugs are resolved. This is fixed in `1.7.3+1` via #100 .

Thanks for filing @dballance . cc @karthik-js @fabricio-sisplan @neelansh-creatorstack

**Comment by scopendo** [comment]
I'm getting an error with 1.7.3+1 – first time install so haven't had earlier versions.

> [!] The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (DatadogSDKCrashReporting)

If tried `pod deintegrate`, `rm Podfile.lock`, `pod install --repo-update` without any improvement.

**Comment by dustin-graham** [comment]
@tshedor I don't think this should be closed. it's broken on the latest build 1.7.3+1.

**Comment by tshedor** [comment]
@scopendo I was able to successfully run this yesterday. I ensured `1.7.3` was installed and `datadog_flutter 0.0.3` was the version in `Podfile.lock`. Your error, however, indicates that there's something in the building in Xcode. I'll most likely need to do something with `s.static_framework` or `s.vendored_libraries`.

I don't have time to reasonably deal with maintenance right now, so I'm rolling back `1.7.3+1` and `1.7.3`. `1.7.3+2` is effectively `.1.72`. cc @Dachmian via #92 .

If anyone can help on Xcode compilation, I'd really appreciate it. Otherwise I'm going to re-close this issue Friday afternoon PST.

**Comment by damir-fell** [comment]
@tshedor We are testing a fix in the mentioned forked repo. If it works we can post a PR here.

**Comment by tshedor** [comment]
@damir-fell looks like we were on [the same track](https://github.com/GetDutchie/datadog_flutter/pull/101/files#diff-f326fbbb2e5f2a953b2277f769784b4c50929c933616ca9677722b38ee1bbd50R21). I'll be merging this in shortly and publishing a pre release.

**Comment by damir-fell** [comment]
@tshedor Awesome, even better.

**Comment by tshedor** [comment]
@damir-fell this is available in `2.0.0-beta.1`
